### DISCUSSION TITLE
how do i send a broadcast message to all the existing bot users?

### DISCUSSION THREAD

**Discussion by ghost** [eternnoir/pyTelegramBotAPI#1589] [Q&A]
i know it will require a file with all the userids , but i dont know any way to do it , i want to send a broadcast message to all the bot users or the users in the file

**Comment by coder2020official** [comment]
Just iterate through a list of tuple of user ids.
Here is an example, but with a usage of antiflood function:
https://gist.github.com/barbax7/6a53fab63bcef592a56ba9edefde6048
